docs/contributing.md

```diff
@@ -73,10 +86,17 @@ Follow these steps to contribute code:
    pytest -n auto tests/lax_scipy_test.py
    ```
 
+   You can narrow the tests further by using the `pytest -k` flag to match particular test
+   names:
+
+   ```bash
+   pytest -n auto tests/lax_scipy_test.py -k testLogSumExp
+   ```
+
    JAX also offers more fine-grained control over which particular tests are run;
    see {ref}`running-tests` for more information.
 
-7. Once you are satisfied with your change, create a commit as follows (
+8. Once you are satisfied with your change, create a commit as follows (
```

Comment:
Two consecutive 8s here, though it doesn't matter because markdown renumbers when rendering.

Reply:
Fixes